model/task_lifecycle.go

```diff
@@ -214,6 +214,13 @@ func DeactivatePreviousTasks(taskId, caller string) error {
 	if err != nil {
 		return err
 	}
+	if t.IsPartOfDisplay() {
+		grip.Error(message.Fields{
+			"message": "attempted to deactivate an execution task",
+			"task":    t.Id,
+			"stack":   message.NewStack(1, "").Raw(),
```

Comment:
I feel like this should be a feature request a new constructor for stack messages that does "wrap stack" or similar. 

Good to push this, but can you open a make ticket to add the new logging feature.